unit_scaling/tests/test_modules.py

```diff
@@ -74,7 +73,7 @@ def test_linear() -> None:
     assert output.shape == torch.Size([2**8, 2**12])
 
     unit_backward(output)
-    SGD(model.parameters(), lr=10).step()
+    SGD(model.parameters(), lr=1).step()
```

Comment:
Curious as to why this changed?

Reply:
No idea! I reverted the old commit that added 0.75 in the first place, but no idea why that commit had to change 1 -> 10 in the first place. A bunch of nasty stuff had to go into the tests when I originally went to 0.75. I guess this was one of them, though I'm not sure why the lr would have had to go up??

Reply:
All good; not a blocking comment & I wondered if that might be the case!